src/liballoc/sync.rs

```diff
@@ -1234,7 +1234,7 @@ impl Arc<dyn Any + Send + Sync> {
     ///
     /// # Examples
     ///
-    /// ```
+    /// ```rust
```

Comment:
This shouldn't be necessary

Reply:
There are bugs in rust.vim that make it doesn't highlight this snippet without rust specifier. Will remove it.